AdminWebsite/AdminWebsite/ClientApp/src/app/services/device-type.ts

```diff
@@ -20,9 +20,17 @@ export class DeviceType {
     }
 
     isSupportedBrowser(): boolean {
-        const supportedBrowsers = [BROWSERS.CHROME, BROWSERS.MS_EDGE_CHROMIUM, BROWSERS.MS_EDGE, BROWSERS.FIREFOX];
+        const unsupportedBrowsers = [
+            BROWSERS.FB_MESSANGER,
+            BROWSERS.SAMSUNG,
+            BROWSERS.UCBROWSER,
+            BROWSERS.OPERA,
+            BROWSERS.UNKNOWN,
+            BROWSERS.IE,
+            BROWSERS.MS_EDGE
```

Comment:
Shouldn't we be supporting Edge?

Reply:
that's the old edge, we want to support edge chromium.